.github/workflows/check-unlinked-content.js

```diff
@@ -0,0 +1,148 @@
+var fs = require("fs");
```

Comment:
I've added this check as a bit of a hotfix as well. Happy to remove if we feel like we're better off without it - I suspect we'll ultimately want to perform this check in `docs-page` (similar to what we do with `validateFilePaths`).